connect/src/com/telenor/connect/WellKnownAPI.java

```diff
@@ -9,17 +9,17 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import retrofit.Callback;
-import retrofit.http.GET;
-import retrofit.http.Headers;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
 
 public interface WellKnownAPI {
 
-    public static final String OPENID_CONFIGURATION_PATH = "/.well-known/openid-configuration";
+    String OPENID_CONFIGURATION_PATH = ".well-known/openid-configuration/";
```

Comment:
Need to cut the trailing slash for the endpoint:
`".well-known/openid-configuration/"` ->
`".well-known/openid-configuration"`

Reply:
Ah yes, I see the problem.

The path in the baseUrl `"oauth/.well-known/openid-configuration/"` is overridden by the path in the annotation `@GET("/")`

But the baseUrl must end with `"/"` or it will crash when you set it in `RestHelper.buildApi()`
https://github.com/simonnorberg/connect-android-sdk/blob/retrofit2/connect/src/com/telenor/connect/utils/RestHelper.java#L79

See documentation
https://square.github.io/retrofit/2.x/retrofit/retrofit2/Retrofit.Builder.html#baseUrl-okhttp3.HttpUrl-

I changed to have the path in the annotation in WellknownAPI instead. Similar to in ConnectAPI.
`@GET("/oauth/.well-known/openid-configuration")`